### ISSUE TITLE
Netflix not working

### ISSUE THREAD

**Issue by ksajan** [brave/brave-browser#17529]
## Description 

When I click on a video to play a video it redirects me to the issue page where it says " You are having trouble watching Netflix" and shows different compatibility with browsers. I am not sure what that means and what setting should I tweak to get it working. 

## Steps to Reproduce

   1. Go to Netflix website
   2. Open a content
   3. You will be greeted by the above mentioned msg

## Actual result:

<img width="1678" alt="Screenshot 2021-08-17 at 9 32 53 PM" src="https://user-images.githubusercontent.com/31013555/129761337-fba033c7-4e47-467b-84de-038587115033.png">

## Expected result:
It should play the content efforlessly

## Reproduces how often: 

Easily Reproduced

## Brave version (brave://version info)

Brave | 1.28.105 Chromium: 92.0.4515.131 (Official Build) (x86_64)
-- | --
Revision | 6b8d6c56ce21e38a72f7c4becb5abc1fa5134f29-refs/branch-heads/4515@{#1933}
OS | macOS Version 11.5 (Build 20G71)

## Version/Channel Information:

- Can you reproduce this issue with the current release? 
- Can you reproduce this issue with the beta channel? 
- Can you reproduce this issue with the nightly channel? 

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? 
- Does the issue resolve itself when disabling Brave Rewards? 
- Is the issue reproducible on the latest version of Chrome? 

## Miscellaneous Information:

This doesn't resolve even after disabling or enabling widewine settings.

**Comment by ryanbr** [comment]
Hey @ksajan can you test Netflix in private window mode, with no extensions enabled

**Comment by ksajan** [comment]
Yeah, It did work in a private window. why is it like that? Does Netflix doesn't work with ublock or other adblocking services anymore?

**Comment by ryanbr** [comment]
Hmm okay, can you disable all extensions, then retry Netflix again (in normal window mode). @ksajan

**Comment by ksajan** [comment]
@ryanbr I did it worked perfectly. And after checking with each extension individually. I found out that even if i have VPN extension installed Netflix doesn't work. But on the hand while on private mode with same VPN it works. I know it out of the scope of this issue but like to understand what changes when you go private mode that same netflix work and normal it doesn't. Is it like in normal mode netflix can access what extension i have installed and behave in certain way and in private mode we don't share those info so it works normally.
